raddb/mods-config/sql/main/mongo/queries.conf

```diff
@@ -56,41 +56,41 @@ sql_user_name = "%{User-Name}"
 # { "attribute" : "Cache-TTL", "value" : 1000, "op" : ":=" }
 #
 authorize_check_query = "db.mobiles.aggregate([ \
-	{ \
-	  '$match': { \
-	    'calling_station_id': '%{Calling-Station-id}', \
-	    'auth_blocked': 'false' \
-	  } \
-	}, \
-	{ \
-	  '$addFields': { \
-	    'attributes.User-Name': '$usr', \
-	    'attributes.ClearText-Password': '$pwd', \
-	    'attributes.Cache-TTL': '$ttlcache', \
-	    'attributes.Enable-Roaming': '$roaming', \
-	    'attributes.Pool-Name': '$pool_name' \
-	   } \
- 	}, \
-	{ 
-	  '$project': { \
-	    'calling_station_id': 1, \
-	    'attributes': { \
-	      '$objectToArray': '$attributes' \
-	    } \
-	  } \
-	}, \
-	{ \
-	  '$unwind': '$attributes' \
-	}, \
-	{ \
-	  '$project': { \
-	    '_id': 0, \
-	    'attribute': '$attributes.k', \
-	    'value': '$attributes.v', \
-	    'op': ':=' \
-	  } \
-	} \
-    ])" \
+    { \
```

Comment:
Can you drop these changes from the patch?  They seem to be just whitespace changes with nothing else.

We try to use tabs everywhere in the config files instead of many spaces.

Reply:
Ok, I fix all query with tabs.

Reply:
it's a bit unsupportable a mixed solution with tabs and spaces, now are used only tabs.

I found also some more fixes for sqlippool queries and some comma issues. Let me know if is ok now. thanks

Reply:
Maybe could be more correct divide commit between fix and indentation.